e2e/03-intraledger-flow.e2e.spec.ts

```diff
@@ -120,7 +120,6 @@ describe("Username Payment Flow", () => {
   it("Click Next again", async () => {
     const nextButton = await $(selector(LL.common.next(), "Button"))
     await nextButton.waitForDisplayed({ timeout })
-    await nextButton.waitForEnabled()
```

Comment:
I believe you can removed `waitForDisplayed`, because if you wait for something to be enabled it will also be displayed. However, just because something is displayed does not mean its enabled, so I think its important to keep the `waitForEnabled`.